pkg/k8shandler/logstore.go

```diff
@@ -120,7 +123,6 @@ func (cluster *ClusterLogging) newElasticsearchCR(elasticsearchName string) *ela
 		Roles:        []elasticsearch.ElasticsearchNodeRole{"client", "data", "master"},
 		NodeCount:    cluster.Spec.LogStore.NodeCount,
 		NodeSelector: cluster.Spec.LogStore.NodeSelector,
-		Resources:    *resources,
```

Comment:
Did we lose this?  It looks like we update to include a default cpuRequest.  Is `esNode` the common defaults?

Reply:
we didn't, but I'd rather us specify it just at the common node level than at both the individual node and the common node.

Reply:
+1 Bigly